tests/test_money.py

```diff
@@ -42,3 +42,45 @@ def test_round():
 def test_configurable_decimal_number():
     # Override default configuration per instance
     assert str(Money("10.543", "USD", decimal_places=3)) == "$10.543"
+
+
+def test_add_decimal_places():
+    one = Money(1.0000, "USD", decimal_places=4)
+    two = Money(2.000000005, "USD", decimal_places=10)
```

Comment:
Generally, I'd avoid creating `Money` instances from `float` across the codebase I suggest using strings instead

Reply:
You're absolutely right there, changed it to strings. Was a bit lenient in the tests as DefaultMoney puts it in a string in the constructor...